### PR TITLE
Added Debian Linux support

### PR DESCRIPTION
**Pull request by sofusalbertsen**
Had to manually move png's to /build/icons/ in order to have the right icon on the Deb package.

### CODE REVIEW

**Comment by sofusalbertsen** [comment]
And added a non-existing email to author part. You are welcome to change that to a real one if you want.

**Comment by emmaarfelt** [comment]
Awesome! Can you elaborate on new lines added to package.json? :-)

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "0.1.2",
   "license": "MIT",
   "description": "Autoreply for Messenger",
-  "author": "Emma Arfelt",
+  "author": "Emma Arfelt <trordu@selv.dk>",
```
Comment (sofusalbertsen) [inline package.json:6]: An email is needed in order to the debian package system to accept the package (you need to know the maintainer)
https://www.debian.org/doc/manuals/debian-faq/ch-pkg_basics.en.html#s-deb-format

File: package.json [hunk 2]
```diff
@@ -31,6 +31,9 @@
     "win": {
       "target": "NSIS",
       "icon": "build/icon.ico"
+    },
+    "linux": {
+      "target": "deb"
```
Comment (sofusalbertsen) [inline package.json:36]: This section tells the `electron-builder build` command to produce a debian package as well as dmg and exe.
Ran it through the docker image: https://hub.docker.com/r/electronuserland/electron-builder/
And it produced a debian package that I could install on Ubuntu (16.04)

**Comment by sofusalbertsen** [comment]
@emmaarfelt done, see the files changed :)

**Pull request merged**